mlir/include/mlir/Dialect/Linalg/TransformOps/LinalgTransformOps.td

```diff
@@ -1819,7 +1819,7 @@ def TileUsingForOp : Op<Transform_Dialect, "structured.tile_using_for",
   let arguments = (ins TransformHandleTypeInterface:$target,
                    Variadic<TransformParamTypeOrAnyHandle>:$dynamic_sizes,
                    DefaultValuedOptionalAttr<DenseI64ArrayAttr, "{}">:$static_sizes,
-                   DefaultValuedOptionalAttr<DenseI64ArrayAttr, "{}">:$interchange,
+                   DefaultValuedOptionalAttr<I64ArrayAttr, "{}">:$interchange,
```

Comment:
Why are we losing the "Dense" aspect here?

Reply:
For this PR, we wanted to stay compatible to the old assembly format which printed as `{interchange = [1,2,3]}`. I think this format (just square brackets) would not have worked with the `DenseAttr`.

Reply:
Let's not do that: `I64ArrayAttr` is a legacy construct, keeping the more modern `DenseI64ArrayAttr` seems preferable and I rather not mixup inherent attribute here anyway.